### PR TITLE
9 new arithmetic functions

### PR DESCRIPTION
**Pull request by sebastialonso**
Last days I've been working in a more mathematical project and I developed this functions. I thought they could be useful in this library for someone in the same place.

Tests included.
Cheers.

### CODE REVIEW

File: is.js [hunk 1]
```diff
@@ -296,6 +296,55 @@
...
+        return is.negative(numb) || numb === 0;
+    }
+
+    // is a given number non-negative?
+    is.nonNegative = function(numb) {
+        return is.positive(numb) || numb === 0;
+    }
+
+    // is a given number perfect square?
+    // Source https://en.wikipedia.org/wiki/Triangular_number
```
Comment (ironmaniiith) [inline is.js]: Don't you think line should be there above is.traingular.?

Comment (sebastialonso) [inline is.js]: Yep, it's fixed in [latest commit](https://github.com/arasatasaygin/is.js/pull/194/commits/5d158be48659441e93ba0501d47fc8d46b3240b8)

Comment (ironmaniiith) [inline is.js]: :+1:

**Comment by jdalton** [comment]
Thanks!

We'll keep an eye on [the popularity of the request](https://github.com/arasatasaygin/is.js/blob/master/CONTRIBUTING.md#feature-requests).

**Pull request closed without merging**